### DISCUSSION TITLE
Understanding of Affine Transforms

### DISCUSSION THREAD

**Discussion by rousseau** [Project-MONAI/MONAI#6979] [Q&A]
Hi all,

I have many questions about the use of affine transformations in MONAI and I can't find clear documentation about the definition of the transformations, especially in relation to the definitions in ITK (which uses world coordinates) or in PyTorch (only in the image coordinates). Which coordinate system is used in MONAI? How to switch from ITK to Pytorch for instance?
Thanks in advance.

**Comment by wyli** [comment]
in case you haven't seen these, there are some initial work on this topic: https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/itk_torch_bridge.py

example from ITKElastix: https://github.com/InsightSoftwareConsortium/ITKElastix/blob/main/examples/ITK_Example18_MONAI_affine_Elastix_nonlinear.ipynb


also https://github.com/InsightSoftwareConsortium/ITKElastix/pull/193/files

**Comment by rousseau** [comment]
Thanks for the link !

Is there an explanation (like that : https://nipy.org/nibabel/coordinate_systems.html) about MONAI coordinates system ?
